Driverless AI/get-started-with-open-source-custom-recipes-tutorial/get-started-with-open-source-custom-recipes-tutorial.md

```diff
@@ -1,4 +1,4 @@
-# Get Started with Open Source Custom Recipes Tutorial
+# Get Started with OpenSource Custom Recipes Tutorial
```

Comment:
The company uses "Open Source" instead of "OpenSource." We should try to be consistent with that or discuss this with the team and see what they think about that

Reply:
@Castro38 @james94 Thoughts?

Reply:
@5675sp, we need to leave the title as is "Get Started with Open Source Custom Recipes Tutorial". This title update was given by Tom K.